src/checkbox/tests/functional/Checkbox.ts

```diff
@@ -0,0 +1,117 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=checkbox')
+		.setFindTimeout(5000);
+}
+
+function nthCheckbox(n: number) {
+	return `fieldset label:nth-of-type(${n}) input[type="checkbox"]`;
+}
+registerSuite({
+	name: 'Checkbox',
+
+	'checkbox should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector(nthCheckbox(1))
+		// as per mwistrand, `isDisplayed` is broken in Safari 10, commented out for now
```

Comment:
Same note here, let's either find an external issue or create one in meta for fixing in the future when Safari driver improves.